app/main.py

```diff
@@ -2,0 +1,54 @@
+class SoftwareEngineer:
+    def __init__(self, name: str) -> None:
+        self.name = name
+        self.skills = []
+
+    def learn_skill(self, skill: str) -> None:
+        self.skills.append(skill)
+
+
+class FrontendDeveloper(SoftwareEngineer):
+    FrontendDeveloper_skills = ["JavaScript", "HTML", "CSS"]
+
+    def __init__(self, name: str) -> None:
+        super().__init__(name)
+        for skill in self.FrontendDeveloper_skills:
+            self.skills.append(skill)
+
+    def create_awesome_web_page(self) -> str:
+        print(f"{self.name} is creating a webpage...")
+        return "<h1>Hello world</h1>"
+
+
+class BackendDeveloper(SoftwareEngineer):
+    BackendDeveloper_skills = ["Python", "SQL", "Django"]
```

Comment:
Similar to the `FrontendDeveloper` class, `BackendDeveloper_skills` should be accessed as a class attribute using `self.__class__.BackendDeveloper_skills` in the `__init__` method.